packages/maps/src/lib/themes/bounds.js

```diff
@@ -1,16 +1,53 @@
-export const GREATER_LONDON_ZOOM = 11;
-export const GREATER_LONDON_CENTER = [0.05, 51.5151];
+export const centerOfBounds = (bounds) => {
+	const midpoint = (min, max) => max - (max - min) / 2;
+	return [midpoint(bounds[0][0], bounds[1][0]), midpoint(bounds[0][1], bounds[1][1])];
+};
+
+// scaleBounds returns a new bounds that scales the input bounds up by the
+// specified amount { x, y } with the center as the origin.
+//
+// Given the amounts:
+// - { x: 0.1, y: 0.1 } then both width and height will be 10% bigger;
+// - { x: 0.5, y: 1 } then width will be 50% bigger and height 100% bigger;
+// - { x: 2, y: 0 } then width will be 200% bigger and height left unchanged;
+// - etc.
+export const scaleBounds = (bounds, amount = { x: 0, y: 0 }) => {
```

Comment:
I'd expect a `scale` function to take scale factors, rather than `(1 - scale-factor)` (this is, for example, what the CSS `scale()` function accepts).

Reply:
I've updated the API & implementation to use scale factors. I also added some simple tests.